sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -96,6 +96,7 @@ func main() {
 	flag.StringVar(&tracing, "tracing", "", "Emit tracing to a Zipkin-compatible tracing endpoint")
 	flag.String("virtualenv", "", "[obsolete] Virtual environment path to use")
 	flag.String("root", "", "[obsolete] Project root path to use")
+	flag.String("typechecker", "", "[obsolete] Use a typechecker to type check")
```

Comment:
Why is the flag marked as obsolete?

Reply:
Passing runtime flags as command line parameters to the language hosts is a deprecated form of passing them. You'll note that main doesn't do anything with these flags.
BUT the engine still sends these flags to language hosts for the time being as a back-compat workaround till all the hosts are updated, so the parser needs to recognize them so it doesn't fail on startup with "unrecognized argument".